### PR TITLE
add new target JHEMCUF745 GF30F745-MPU

### PR DESCRIPTION
**Pull request by pablitous**
Well, i bought two of these boards and I decided i want to try inav in a 7 inch LR / Cinematic drone.
So i decided to learn and after 3 days of reading, learning, datasheeting, testing and some help from my friend [axelfenner ](https://github.com/axelfenner ) I finally made it work with full functionality. At least all i tested worked.
It would be great if someone else could try it too
Let me know anything if I have to change it.
Im gonna leave here the hex file if anyone want to try it (its based on inav 7.1-RC1). or you can try to build it yourself for any other inav version
[inav_7.1.0_JHEMCUF745.zip](https://github.com/iNavFlight/inav/files/14620097/inav_7.1.0_JHEMCUF745.zip)

This is the board:
S219ef091778a448e9b28001455d28718P

and these are some photos from testing, telemetry working, gps, serialrx with elrs900 and a beeper:
375CF4FA-ACE3-423C-92AE-61C26E67B1C2

IMG_9755

IMG_9754

### CODE REVIEW

File: src/main/target/JHEMCUF745/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+target_stm32f745xg(JHEMCUF745)
```
Comment (mmosca) [inline src/main/target/JHEMCUF745/CMakeLists.txt]: Can you add SKIP_RELEASES to the target.

**Comment by mmosca** [comment]
Are you affiliated with JHEMCU? 

Please check https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md


## New target additions

New targets are accepted into INAV code if any of the following conditions is satisfied:

1. Requester is a manufacturer of the hardware or an affiliated community member. Requester provides specs, schematics and production samples are provided to at least two core developers. In this case the new target becomes part of official INAV release.

2. Requester is a community member not affiliated with a manufacturer of the hardware. Requester provides board samples to at least one core developer and the target is already supported in official Cleanflight or Betaflight firmware. In this case the new target may or may not become part of official release based on the decision made by core developers.

3. The new target must meet the following minimal requirements:

  * On-board sensors include at least the IMU (gyroscope + accelerometer)
  * At least 2 hardware serial ports are available with both TX and RX pins
  * At least 512K of firmware flash memory and at least of 64K of RAM available
  * At least one I2C bus broken out (SCL and SDA pins) and not shared with other functions

File: src/main/target/JHEMCUF745/CMakeLists.txt (hunk 1 above)
Comment (pablitous) [reply src/main/target/JHEMCUF745/CMakeLists.txt]: sure

**Comment by pablitous** [comment]
> Are you affiliated with JHEMCU?
> 
> Please check https://github.com/iNavFlight/inav/blob/master/docs/policies/NEW_HARDWARE_POLICY.md
> 
> ## New target additions
> New targets are accepted into INAV code if any of the following conditions is satisfied:
> 
> 1. Requester is a manufacturer of the hardware or an affiliated community member. Requester provides specs, schematics and production samples are provided to at least two core developers. In this case the new target becomes part of official INAV release.
> 2. Requester is a community member not affiliated with a manufacturer of the hardware. Requester provides board samples to at least one core developer and the target is already supported in official Cleanflight or Betaflight firmware. In this case the new target may or may not become part of official release based on the decision made by core developers.
> 3. The new target must meet the following minimal requirements:
> 
> * On-board sensors include at least the IMU (gyroscope + accelerometer)
> * At least 2 hardware serial ports are available with both TX and RX pins
> * At least 512K of firmware flash memory and at least of 64K of RAM available
> * At least one I2C bus broken out (SCL and SDA pins) and not shared with other functions

im not related to JHEMCU, i will try to contact them, so maybe they can send a few board to you guys, and make this oficial.
ty

**Comment by mmosca** [comment]
FYI; JHEMCU got in touch, so an official target for this and 2 more fcs is likely going to be coming soon to INAV.

**Comment by pablitous** [comment]
> FYI; JHEMCU got in touch, so an official target for this and 2 more fcs is likely going to be coming soon to INAV.

niiiiiiice. Glad to hear good news.

**Comment by jhemcu** [comment]
Hello, @pablitous  thank you very much for your efforts. I will soon upload the target file corresponding to the board and send the board to community members for review.

**Comment by jhemcu** [comment]
@pablitous Hello, I have uploaded the target file. You can help review and point out the problems. I have tested it carefully....
https://github.com/iNavFlight/inav/pull/9807

**Comment by mmosca** [comment]
Closing, as jhemcu is submitting an official target in #9807

**Pull request closed without merging**